config/gitlab.yml.example

```diff
@@ -80,6 +80,11 @@ production: &base
       wiki: true
       snippets: false
 
+      ## Default web hooks settings
```

Comment:
@randx We are trying to move all the settings to the settings page right? So no new settings should be added to gitlab.yml?